packages/decentraland-ecs/src/decentraland/Components.ts

```diff
@@ -1135,4 +1149,12 @@ export class VideoTexture extends ObservableComponent {
 
     return super.toJSON()
   }
+
+  update(videoEvent: IEvents['videoEvent']) {
+    if (videoEvent.videoClipId == this.videoClipId) {
+      this.data['status'] = (videoEvent.videoStatus as VideoStatus) || VideoStatus.NONE
+      this.data['videoLength'] = videoEvent.totalVideoLength
+      this.data['position'] = videoEvent.currentOffset
```

Comment:
```suggestion
      this.status = (videoEvent.videoStatus as VideoStatus) || VideoStatus.NONE
      this.videoLength = videoEvent.totalVideoLength
      this.position = videoEvent.currentOffset
```

Reply:
Sorry for missing this in the previous review, but `data` only works with the decorators

Reply:
Something you can do is using getters like this:

```ts
class MyComponent {
  private _status: number = DEFAULT_STATUS

  get status(): number {
    return this._status
  }

  // use @internal to strip this declaration from the final .d.ts
  // @internal
  update({ status }) {
    this._status = status
  }
}

```